wikidp/templates/snippets/contribute_input_templates.html

```diff
@@ -12,35 +12,69 @@
         </div>
     </script>
 
-    <script id="wikidp-added-claim-li" type="text/template">
```

Comment:
Same goes for this file, the majority of the diff is moving towards react/vue-friendly html formatting in case we ever want to move things over.
The only functional change is the addition of the `{{#references}}` loop at the bottom